frontend/packages/local-storage-operator-plugin/src/components/tables/select-nodes-table.tsx

```diff
@@ -61,7 +68,7 @@ const getRows: GetRows = (
         title: roles.join(', ') ?? '-',
       },
       {
-        title: getZone(node) || '-',
+        title: node.metadata.labels?.['failure-domain.beta.kubernetes.io/zone'] ?? '-',
```

Comment:
Why are we removing the usage of getZone?

Reply:
Possibly by miss.
Will revert.